test/unit/init/cluster/ClusterManager.test.ts

```diff
@@ -0,0 +1,102 @@
+import cluster from 'cluster';
+import EventEmitter from 'events';
+import { cpus } from 'os';
+import { ClusterManager } from '../../../../src';
+import * as LogUtil from '../../../../src/logging/LogUtil';
+
+jest.mock('cluster');
+jest.mock('os', (): any => ({
+  ...jest.requireActual('os'),
+  cpus: jest.fn().mockImplementation((): any => [{}, {}, {}, {}, {}, {}]),
+}));
+
+describe('A ClusterManager', (): void => {
+  const emitter = new EventEmitter();
+  const workerEmitter = new EventEmitter();
+  const mockCluster = jest.requireMock('cluster');
+  const mockWorker = {
+    on: jest.fn().mockImplementation(workerEmitter.on),
+    emit: jest.fn().mockImplementation(workerEmitter.emit),
+    process: { pid: 666 },
+  };
```

Comment:
I don't see the point of this class since none of the tests use the mock part of these functions. Something like this should already be enough then
```ts
  const workerEmitter = new EventEmitter() as CorrectWorkerTypeHere;
  workerEmitter.process = { pid: 666 };
```